### PR TITLE
Add FPS measurement

### PR DESCRIPTION
**Pull request by sigvef**

### CODE REVIEW

File: client/src/bootstrap.js [hunk 1]
```diff
@@ -62,19 +66,27 @@ function loop() {
   dt += (t - old_time);
   old_time = t;
   songTime = mm.music.currentTime;
-  while (dt > MS_PER_FRAME) {
+  while (dt >= MS_PER_UPDATE_FRAME) {
     sm.update();
     mm.update();
     MOUSE.scrollX = 0;
     MOUSE.scrollY = 0;
     tick++;
```
Comment (lionleaf) [inline client/src/bootstrap.js:74]: tick should be obsolete now, right?  (it's at least referenced when sending a state)

Comment (sigvef) [inline client/src/bootstrap.js:74]: Nope, tick is used to synchronize state between the server and client.

Comment (lionleaf) [inline client/src/bootstrap.js:74]: Ah, this is client side loop!! :D

**Comment by sigvef** [comment]
ping

**Pull request merged**